scripts/util/create-DR.js

```diff
@@ -44,7 +43,7 @@ const addressNotFound = (address) => {
   process.exit(1);
 };
 
-const createAndActivateDR = async (path, createOnly, activateOnly) => {
+const createDR = async (path) => {
```

Comment:
Since this method name is shorter now, I believe we should call it `createDisputeResolver`. We don't want to abbreviate unless we have to.